test cases/cmake/26 dependency fallback/subprojects/cmMod.wrap

```diff
@@ -0,0 +1,5 @@
+[wrap-file]
+method = cmake
+
+[provide]
+cmModLib++ = cmModLib___dep
```

Comment:
What is the technical reason for this?

Reply:
> Instead, the `dep_name = <target_name>_dep` syntax should be used, where `<target_name>` is the name of a CMake library with all non alphanum characters replaced by underscores `_`

target_name is `cmModLib++`, replace non alphanum with underscore gives `cmModLib__` and append `_dep` gives `cmModLib___dep`, that's the name of the variable in the generated AST.